views/includes/page-templates/trust-historical-bo.html

```diff
@@ -51,15 +52,15 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ pageParams.title }}</h1>
           {
             value: pageData.trusteeType.LEGAL_ENTITY,
             text: "Legal Entity",
-            checked: formData.type == pageData.trusteeType.LEGAL_ENTITY,
+            checked: (formData.type | lower) === (pageData.trusteeType.LEGAL_ENTITY | lower),
```

Comment:
Wondering why this change is needed for Redis?

Reply:
This was needed because the mixed-case type value was not being deciphered as being equal to the all-caps type value. But with the mapping done as discussed in the huddle, we might not need this. Thanks for pointing it out though.

Reply:
👍 yes, hopefully not required when we use that common mapping code.

Reply:
Agreed. We'll not need this after the mapping is applied, so will refactor it back to what it was originally.